build.gradle

```diff
@@ -592,6 +598,30 @@ project('helloworld') {
 
 }
 
+project('helloworld-groovy') {
+	description = 'Hello World Sample for Groovy Developers'
+
+	apply plugin: 'groovy'
+
+	dependencies {
+		api 'org.springframework.integration:spring-integration-core'
+		api "org.apache.logging.log4j:log4j-core:$log4jVersion"
+		implementation "org.apache.groovy:groovy:$groovyVersion"
+		api "org.springframework.integration:spring-integration-groovy"
```

Comment:
I believe we need only this dependency.
Everything else is pre-configured as a common dep, or comes as transitive.